loops.js

```diff
@@ -16,9 +18,13 @@
 // loop...
 // loop...
 // klaar!
- // ==========================================
-
-
+// ==========================================
+for (let x = 0; x < 4; x++) {
+    console.log("loop...");
+}
+if (x => 4) {
+    console.log("klaar");
+}
```

Comment:
Volgens mij wordt vaak de i genomen in plaats van de x (die i staat voor index), maar dat is geen must. Het moet volgens de uitwerking van Nova wel zijn x <5 ipv x <4. (in jouw code wordt de tweede if nooit uitgevoerd omdat x nooit groter of gelijk aan 4 is.